src/NServiceBus.Gateway.AcceptanceTests/NServiceBusAcceptanceTest.cs

```diff
@@ -13,6 +13,10 @@ public abstract class NServiceBusAcceptanceTest
         [SetUp]
         public void SetUp()
         {
+            // Hack: prevents SerializationException ... Type 'x' in assembly 'y' is not marked as serializable.
+            // https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/mitigation-deserialization-of-objects-across-app-domains
+            System.Configuration.ConfigurationManager.GetSection("X");
```

Comment:
didnt we fix this in core?

Reply:
It was fixed in the shipped acceptance tests but the GW only runs it own test so I needed to copy it over

https://github.com/Particular/NServiceBus/blob/ed11623e2d7f6fdedee1b756c800f06606a88bfb/src/NServiceBus.AcceptanceTests/NServiceBusAcceptanceTest.cs#L21